constructor/_schema.py

```diff
@@ -152,7 +152,7 @@ class HashBuildOutput(BaseModel):
     """
 
     model_config: ConfigDict = _base_config_dict
-    hash_: _HashBuildOutputOptions
+    hash: _HashBuildOutputOptions
```

Comment:
It's better to use the `alias` keyword in `Field` so we don't override the `hash()` builtin in this scope (not that it matters here, but it's a better practice).